ssl/ssl_test.cc

```diff
@@ -5160,9 +5166,10 @@ class MultipleCertificateSlotTest
     return std::get<1>(GetParam());
   }
 
-  void StandardCertificateSlotIndexTests(SSL_CTX *client_ctx,
-                                         SSL_CTX *server_ctx,
-                                         std::vector<uint16_t> sigalgs) {
+  const void StandardCertificateSlotIndexTests(SSL_CTX *client_ctx,
```

Comment:
Are any additional tests required to specifically account for the changes in when the OCSP callbacks are handled? What about validating functionality in AWS-LC that requires the public key to be stored (ensuring the right key gets cached etc)?

Reply:
* The OCSP callbacks are already tested in our SSL runner tests. If the callback isn't configured or called in the handshake, the tests will fail correspondingly.
https://github.com/aws/aws-lc/blob/a818ddf61ed8bbf41516fbd0b8dabd64e166a9b1/ssl/test/test_config.cc#L1577-L1584

* Great point, I'll have to add tests for that. Also thinking if there's a better way to help us enforce that we reload the public key whenever the certificate is updated.